Add identifier structure to `g7:NOTE` (#213)
FamilySearch/GEDCOM: specification/gedcom-03-datamodel.md

A catch-all location for information that does not fully fit within other structures.
It may include research notes, additional context, alternative interpretations, reasoning, and so forth.

A shared note record may be pointed to by multiple other structures. Shared notes should only be used if editing the note in one place should edit it in all other places.

> If each instance of the note may be edited separately, a `NOTE` should be used instead.

Author: ```suggestion
If any instance of the note is edited separately from the others, a `NOTE` should be used instead.
```
